### ISSUE TITLE
Hello,can you help me?

### ISSUE THREAD

**Issue by longmalongma** [CyberZHG/keras-targeted-dropout#2]
import keras
from keras_targeted_dropout import TargetedDropout

model = keras.models.Sequential()
model.add(TargetedDropout(input_shape=(None, None), drop_rate=0.4, target_rate=0.4))
model.compile(optimizer='adam', loss='mse')
model.summary()

I want to know where can I add these code ?

**Comment by CyberZHG** [comment]
https://github.com/CyberZHG/keras-targeted-dropout/blob/master/README.zh-CN.md
